frontend/packages/console-demo-plugin/src/plugin.tsx

```diff
@@ -110,6 +110,27 @@ const plugin: Plugin<ConsumedExtensions> = [
       landingPageURL: '/search',
     },
   },
+  {
+    type: 'NavItem/Href',
+    properties: {
+      perspective: 'test',
+      componentProps: {
+        name: 'Test Search',
+        href: '/search',
+      },
+    },
+  },
+  {
+    type: 'NavItem/ResourceCluster',
+    properties: {
+      perspective: 'test',
+      section: 'Advanced',
```

Comment:
If a section is specified but not already present in the nav menu, it will be added as part of nav item processing.

Another approach is to have a separate extension for adding sections, with the ability to specify perspective name. What do you think?

Reply:
Dev console needs nav links without sections.
To truly benefit we would have to separate the concept of a navigational action, from the structure of a nav. I've suggested this previously and maybe something we can discuss for a future refactoring PR.

Reply:
Agreed, we can discuss it and improve in future.